source/acquaman/AMGenericStepScanConfiguration.cpp

```diff
@@ -283,16 +283,18 @@ void AMGenericStepScanConfiguration::setRegionOfInterestBoundingRange(AMRegionOf
 
 void AMGenericStepScanConfiguration::setI0(const AMDetectorInfo &info)
 {
+	bool validInfo = (info.name() != "Invalid detector");
```

Comment:
I know it's not a big deal. But personally, I would prefer to have a helper function in AMDetectorInfo called `isValidInfo()`. The reason for that is, the caller of AMDetectorInfo won't need to know how the valid/invalid name is definition.